CONTRIBUTING.md

```diff
@@ -1,11 +1,11 @@
-# Contributing to OpenWhisk
+# Contributing to OpenWhisk catalog
 
 We welcome contributions, but request you follow these guidelines.
```

Comment:
but request you follow these guidelines =>
but request you to follow these guidelines or but aks you to follow these guidelines